### PR TITLE
Remove/Replace Balanced from Credit card page

### PR DESCRIPTION
**Pull request by kaguillera**
Failed test for the removal of Balanced from the Credit-card page

### CODE REVIEW

**Comment by whit537** [comment]
Partially addresses #3514.

**Comment by kaguillera** [comment]
Please review. @whit537, @rohitpaulk

**Comment by rohitpaulk** [comment]
Added few commits, rebased on master. cc: @kaguillera

**Comment by kaguillera** [comment]
Removed Balanced TOS and Privacy Policy from the spt and md files since we are no longer using them. Replaced the information with Braintree's not sure if what I placed is legally appropriate since not all of the functions performed by Balanced are being done by Braintree.
Please review @whit537

**Comment by whit537** [comment]
Rebased on master.

**Comment by whit537** [comment]
I would like to see this PR not affect our test fixtures. What's requiring network access here?

Secondly, I'm not sure the failing test where we check that certain text is no longer on the page is the best idea, because there's an infinity of text we could check for the absence of. I think it makes more sense to test for the *presence* of Braintree.

The legalese needs to be proofread ...

**Comment by whit537** [comment]
@kzisme in #3769:

> When reviewing a users routes page
>
https://gratipay.com/~kzisme/routes/
>
There is still an image showing that payments are 
capture
>
Is there a preferred image that could replace the existing or would it be okay to keep it around?

**Comment by whit537** [comment]
> Is there a preferred image that could replace the existing or would it be okay to keep it around?

@kzisme To answer your question we definitely want to get Balanced off the site since we're not processing with them anymore (last thing is #3539). There's some issues with this PR, I'm planning to get to it at some point post-[Pivot](https://github.com/gratipay/gratipay.com/milestones/Pivot), but if anyone else wants to review/work on it before then, go for it! :-)

**Comment by kzisme** [comment]
No worries I was just clicking around on my own account and noticed that and figured it would/could be documented. :smile:

**Comment by whit537** [comment]
@mattbk Would you be willing and able to pick up with this one? It's on the Balanced Shutdown milestone, which is due soon.

**Comment by mattbk** [comment]
I think so, I assume I have to learn how to rebase?

**Comment by whit537** [comment]
@mattbk And resolve conflicts. If that's too much trouble you could close this and start over.

**Comment by mattbk** [comment]
Okay, I'll look into it this week.

**Comment by whit537** [comment]
Awesome, thanks. :)

**Comment by mattbk** [comment]
I rebased on master (I think).  The only conflict was on test_pages.py, so I did not merge that file.  I'm not really sure what the tests do, so I guess I'm looking into that.

**Comment by mattbk** [comment]
For me to make progress on this I'm going to have to diff test_pages.py and see how things are supposed to work.

**Comment by mattbk** [comment]
Test failed because I didn't indent right.

**Comment by mattbk** [comment]
Okay, can someone review?  I got Travis to work :-)

**Comment by nathany** [comment]
Does this remove all the last occurrences of "Balanced"?

balanced1

balanced

**Comment by mattbk** [comment]
Yes.

**Comment by whit537** [comment]
@nathany Up for reviewing it? :)

File: www/about/policies/terms-of-service.spt [hunk 1]
```diff
@@ -238,17 +238,15 @@ responsible for the activity.</li>
 <li><p>Gratipay&#39;s Service Providers</p>
 
 <ol>
-<li>Gratipay uses Pound Payments Inc., DBA Balanced (&ldquo;Balanced&rdquo;) as a third
+<li>Gratipay uses Pound Payments Inc., Braintree (&ldquo;Braintree&rdquo;) as a third
```
Comment (nathany) [inline www/about/policies/terms-of-service.spt]: Pound Payments Inc.?

Comment (kzisme) [inline www/about/policies/terms-of-service.spt]: I believe Pound Payment is the same thing as Balanced based off of [this](https://poundpay.com/terms/marketplaceagreement) - I'm sure @whit537 has the most correct answer :)

Comment (whit537) [inline www/about/policies/terms-of-service.spt]: Yeah, Pound Payments was their old name.

**Comment by whit537** [comment]
Rebased on master, hit a conflict. Why u no see, Travis?

**Comment by whit537** [comment]
Rebased on master. Original HEAD was edbbd862708793ecb591656ebe9439182045f2d3.

**Comment by whit537** [comment]
> Secondly, I'm not sure the failing test where we check that certain text is no longer on the page is the best idea, because there's an infinity of text we could check for the absence of.

**Comment by whit537** [comment]
But I'm reluctant to remove that test now that @mattbk has finally gotten it working. :-)

!m @mattbk

**Comment by whit537** [comment]
Gonna proofread ...

**Comment by whit537** [comment]
> But I'm reluctant to remove that test now that @mattbk has finally gotten it working. :-)

Sorry, @mattbk *did* observe https://github.com/gratipay/gratipay.com/pull/3565#issuecomment-149970855, but the name of the test threw me off. ;-)

**Comment by whit537** [comment]
8252a5b6a521408e9fd00d2f74bd546b08d197f7 closes #3514.

**Comment by whit537** [comment]
I think we're ready here. Anything else @nathany @mattbk @kaguillera?

**Pull request merged**